example/lib/develop/peripheral_list/peripheral_list_screen.dart

```diff
@@ -0,0 +1,129 @@
+import 'dart:io';
+
+import 'package:blemulator_example/develop/model/ble_peripheral.dart';
+import 'package:blemulator_example/develop/peripheral_list/bloc.dart';
+import 'package:blemulator_example/develop/peripheral_list/components/connection_state_view.dart';
+import 'package:blemulator_example/develop/peripheral_list/components/rssi_view.dart';
+import 'package:blemulator_example/develop/util/custom_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class PeripheralListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return _buildTestLayout(context);
+  }
+
+  Widget _buildTestLayout(BuildContext context) {
```

Comment:
that does mean `test` in this context?

Reply:
This is a leftover from the early stage of this class implementation, it's going to be changed.

Reply:
Fixed